args.js

```diff
@@ -65,7 +66,9 @@ module.exports = function parseArgs (args) {
 
   // Merge objects from lower to higher priority
   const parsedArgs = { ...DEFAULT_ARGUMENTS, ...configFileOptions, ...commandLineArguments }
-
+  const isNumber = isNaN(Number(TRUST_PROXY)) === false
+  const isTrue = TRUST_PROXY.toLowerCase() === 'true'
+  const trustProxy = isNumber ? Number(TRUST_PROXY) : isTrue
```

Comment:
this is very hard to read and it does too much logic IMHO

regardless this code, the `parsedArgs` will win if the user set a boolean.. I would hardly understand the behaviour here.

KIS approach:
- one env per boolean
- one env per trust proxy hosts
- one env per trust hop

the priority is the list i wrote when user set multiple env vars